docs/book/checkout.rst

```diff
@@ -0,0 +1,259 @@
+.. index::
+   single: Checkout
+
+Checkout
+========
+
+**Checkout** is a process that begins when the Customer decides to finish their shopping and transform their **Cart** into an **Order**.
+
+Checkout State Machine
+----------------------
+
+The Order Checkout state machine has 5 states available: ``cart``, ``addressed``, ``shipping_selected``, ``payment_selected``, ``completed``
+and a set of defined transitions between them - which are saved as the **checkoutState** of the **Order**.
+
+Besides the steps of checkout, each of them can be done once more. For instance if the Customer changes their mind
```

Comment:
`once more` => `more than once`, the first one suggests you can only do those things twice during the same checkout.